botfront/imports/api/graphql/common/common.types.graphql

```diff
@@ -47,6 +47,7 @@ input EntityInput {
     start: Int
     end: Int
     confidence: Float
+    confidence_entity: Float
```

Comment:
could I get a quick explanation on what this is for? I can't find it referenced anywhere else.

Reply:
WHen I use crfEntityExtractor that's what confidence is named, and Graphql doesn't save if there are unknown properties